### ISSUE TITLE
UI changed / buttons cut off at the bottom

### ISSUE THREAD

**Issue by Aimtu321** [lng2020/GOP3-blackjack-bot#7]
Bot not doing anything. UI change yesterday and the bot probably not recognizing the buttons.

**Comment by Aimtu321** [comment]
![Uploading 1.jpg…]()

**Comment by Aimtu321** [comment]
![1](https://github.com/user-attachments/assets/02cfa4cd-9ea0-4f56-9738-d15e39941dee)

**Comment by Aimtu321** [comment]
Doesnt matter if window mode or fullscreen or different resolution

**Comment by lng2020** [comment]
You should play in a personal room.

**Comment by Aimtu321** [comment]
Yes I know. The screenshot is onyl to show the ui. Its also in private room.

**Comment by lng2020** [comment]
Thanks for the information. I will check it today. Though I haven't play the game anymore lol.

**Comment by Aimtu321** [comment]
Maybe the game devs will fix it. Doesnt look intended. But if its an easy fix for you go ahead :D

**Comment by Aimtu321** [comment]
I made a little test with new images and it worked.

old image:
![bet1k](https://github.com/user-attachments/assets/9b691e3a-e7cd-4224-9360-38f53207383a)

new image:
![bet1k](https://github.com/user-attachments/assets/baf7ee4f-57e6-4488-9004-970c89f50838)

**Comment by Aimtu321** [comment]
Working with these images

![bet25k](https://github.com/user-attachments/assets/eb7852a7-e917-41c0-9c7f-769b090e8d3d)
![bet50k](https://github.com/user-attachments/assets/db700d28-3988-4088-bac5-0bb6ffd5e7ae)
![bet100k](https://github.com/user-attachments/assets/3ed07b3c-3908-4a4b-9402-0d60bfad919b)
![bet200k](https://github.com/user-attachments/assets/a11e0cf8-5b38-4685-ae05-02e56d3b12e5)
![double](https://github.com/user-attachments/assets/4352fc0b-f4d5-48cf-9785-ddbb2324cf2a)
![hit](https://github.com/user-attachments/assets/2cdb22e7-cb30-47ae-9026-6c5803f5d8e5)
![split](https://github.com/user-attachments/assets/8e0aa026-4eba-40d3-914b-add0356d742f)
![stand](https://github.com/user-attachments/assets/8b406786-17d1-4b12-8713-620222b79d53)

**Comment by Aimtu321** [comment]
But still problems letting the bot run in a vm. The bets are working but hit, stand, double and split not working in vm. On the host machine it works. Any idea whats wrong with vm?

**Comment by lng2020** [comment]
I logged in today and found the UI didn't change in my local and the bot can still play. Anything I'm missing?
